qiime2/sdk/plugin_manager.py

```diff
@@ -48,6 +48,7 @@ def __new__(cls):
     def _init(self):
         self.plugins = {}
         self.semantic_types = {}
+        self.importable_formats = {}
```

Comment:
Can you make this a private attribute, `_importable_formats`, and make it accessible via a read-only `importable_formats` property? I realize there are many public attributes already in `PluginManager` but moving forward I'd like to start making this class safer for SDK developers. For example, it'd be bad if an SDK user imported the `PluginManager` and started mutating the set of importable types after instantiation.